test/functional/helpers/extensionViewController.js

```diff
@@ -97,5 +97,9 @@ export default {
   async expectSettings(expectedSettings) {
     const actualSettings = await getSettings();
     await t.expect(actualSettings).eql(expectedSettings);
+  },
+  async expectSettingsToContain(expectedSettings) {
```

Comment:
For some of the tests I needed a method that checked that the settings object contained certain properties. I created a method for that.